api/proto/teleport/legacy/types/types.proto

```diff
@@ -9046,6 +9046,30 @@ message AWSMatcher {
   // This value should match the IAM identity that Teleport Kubernetes Service uses.
   // If this value is empty, the Discovery Service will attempt to set up access for its own identity (self).
   string SetupAccessForARN = 9 [(gogoproto.jsontag) = "setup_access_for_arn,omitempty"];
+  // OrganizationID is the AWS Organization ID to match when discovering accounts.
+  // Required when defining account matchers.
+  string OrganizationID = 10 [(gogoproto.jsontag) = "organization_id,omitempty"];
+  // Accounts contains filter rules for matching AWS accounts under the Organization.
+  // All accounts are matched if no rule is specified.
+  AWSAccountsMatcher Accounts = 11 [(gogoproto.jsontag) = "accounts,omitempty"];
```

Comment:
i am little confused by this filter. it's called `accounts` but only filter allowed is at OU level?

Reply:
It should support more filters in the future.
Eg, excluding or including by account IDs.

I think it is clear, because you usually filter by OUs when trying to filter all the accounts in a given organization.

I'm open to suggestions.

This is the purposed structure:
```yaml
organization_id: o-hello
accounts:
  include:
    ou: ["r-123", "ou-123]
  exclude:
    ou: ["ou-456"]
```

What do you think about
```yaml
accounts:
  organization_id: o-hello
  ou:
    include: ["r-123, "ou-123"]
    exclude: ["ou-456"]
```

We can also spell out the `ou` into `organizational unit`


cc @r0mant because we discussed this before